lib/rmt/cli/mirror.rb

```diff
@@ -79,7 +79,13 @@ def product(*targets)
   protected
 
   def mirror
-    @mirror ||= RMT::Mirror.new(logger: logger, mirror_src: RMT::Config.mirror_src_files?)
+    config = {
+      logger: logger,
+      mirroring_base_dir: RMT::DEFAULT_MIRROR_DIR,
+      mirror_src: RMT::Config.mirror_src_files?,
+      airgap_mode: false
```

Comment:
Was it a bug that we didn't set `airgap_mode` before?

Reply:
It defaults to to false. For the sake of being explicit  we added it here